core/src/main/java/com/google/googlejavaformat/java/LineSeparator.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Copied it from `RemoveUnusedImports.java`. Will use another source for copy & paste.